minio/api.py

```diff
@@ -139,12 +139,11 @@ def __init__(self, endpoint, access_key=None,
 
         # Load CA certificates from SSL_CERT_FILE file if set
         timeout = timedelta(minutes=5).seconds
-        ca_certs = os.environ.get('SSL_CERT_FILE') or certifi.where()
```

Comment:
What issue are you facing here?

Reply:
I have custom http_client, and do not have installed certificates in my setup. So completely valid setup(I trust my private s3) just throws exceptions.

My patch delay certificate load, only if we really create urllib3.PoolManager

Reply:
Can you share what exceptions are you getting?

Reply:
Why? It can be in any range from FileNotFoundError, to (in some cases) error from Python's importlib internals, cause I use frozen python modules for distribution.

Reply:
Makes sense. This info is needed for someone tracking PRs/issues in future.